reference/constraints/Traverse.rst

```diff
@@ -0,0 +1,94 @@
+Traverse
+========
+
+Objects do not validate nested objects by default unless explicitly using this constraint.
```

Comment:
Please add a line break after the first word crossing the 72th character

Reply:
done!